### ISSUE TITLE
Don't log errors when a TS Server plugin is not found in one folder but is eventually resolved

### ISSUE THREAD

**Issue by mjbvz** [microsoft/TypeScript#30106]
**TypeScript Version:**  typescript@3.4.0-dev.20190226

**Search Terms:**
- TS Server plugins
- tsserver.log

**Repo**
1. In VS Code
1. Enable TS Server logging by setting: `"typescript.tsserver.log": "verbose"`
1. Install the tslint plugin: https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-typescript-tslint-plugin
1. Open a ts file

**Bug**
Although the plugin is eventually resolved, we end up logging a number of errors in TS Server log while searching for it. These errors are distracting and suggest to users that something is going wrong, whereas they are actually expected. Here's an example error:

```
Failed to load module 'typescript-tslint-plugin': Error: Could not resolve JS module 'typescript-tslint-plugin' starting at '/Users/matb/.vscode-insiders/extensions/jpoissonnier.vscode-styled-components-0.0.25/node_modules'. Looked in: /Users/matb/.vscode-insiders/extensions/jpoissonnier.vscode-styled-components-0.0.25/node_modules/typescript-tslint-plugin/package.json, /Users/matb/.vscode-insiders/extensions/jpoissonnier.vscode-styled-components-0.0.25/node_modules/typescript-tslint-plugin.js, /Users/matb/.vscode-insiders/extensions/jpoissonnier.vscode-styled-components-0.0.25/node_modules/typescript-tslint-plugin.jsx, /Users/matb/.vscode-insiders/extensions/jpoissonnier.vscode-styled-components-0.0.25/node_modules/typescript-tslint-plugin/index.js, /Users/matb/.vscode-insiders/extensions/jpoissonnier.vscode-styled-components-0.0.25/node_modules/typescript-tslint-plugin/index.jsx, /Users/matb/.vscode-insiders/extensions/node_modules/typescript-tslint-plugin/package.json, /Users/matb/.vscode-insiders/extensions/node_modules/typescript-tslint-plugin.js, /Users/matb/.vscode-insiders/extensions/node_modules/typescript-tslint-plugin.jsx, /Users/matb/.vscode-insiders/extensions/node_modules/typescript-tslint-plugin/index.js, /Users/matb/.vscode-insiders/extensions/node_modules/typescript-tslint-plugin/index.jsx, /Users/matb/.vscode-insiders/node_modules/typescript-tslint-plugin/package.json, /Users/matb/.vscode-insiders/node_modules/typescript-tslint-plugin.js, /Users/matb/.vscode-insiders/node_modules/typescript-tslint-plugin.jsx, /Users/matb/.vscode-insiders/node_modules/typescript-tslint-plugin/index.js, /Users/matb/.vscode-insiders/node_modules/typescript-tslint-plugin/index.jsx, /Users/matb/node_modules/typescript-tslint-plugin/package.json, /Users/matb/node_modules/typescript-tslint-plugin.js, /Users/matb/node_modules/typescript-tslint-plugin.jsx, /Users/matb/node_modules/typescript-tslint-plugin/index.js, /Users/matb/node_modules/typescript-tslint-plugin/index.jsx, /Users/node_modules/typescript-tslint-plugin/package.json, /Users/node_modules/typescript-tslint-plugin.js, /Users/node_modules/typescript-tslint-plugin.jsx, /Users/node_modules/typescript-tslint-plugin/index.js, /Users/node_modules/typescript-tslint-plugin/index.jsx, /node_modules/typescript-tslint-plugin/package.json, /node_modules/typescript-tslint-plugin.js, /node_modules/typescript-tslint-plugin.jsx, /node_modules/typescript-tslint-plugin/index.js, /node_modules/typescript-tslint-plugin/index.jsx
    at Object.resolveJSModule (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:26487:19)
    at Object.sys.require (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:128711:45)
    at Function.Project.resolveModule (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:122024:35)
    at /Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:122699:36
    at Object.firstDefined (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:238:26)
    at ConfiguredProject.Project.enablePlugin (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:122698:41)
    at _loop_1 (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:122683:32)
    at ConfiguredProject.Project.enableGlobalPlugins (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:122688:25)
    at ConfiguredProject.enablePluginsWithOptions (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:122925:22)
    at ProjectService.loadConfiguredProject (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:124101:25)
    at ProjectService.createAndLoadConfiguredProject (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:124057:22)
    at ProjectService.createLoadAndUpdateConfiguredProject (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:124061:36)
    at ProjectService.openClientFileWithNormalizedPath (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:124717:44)
    at IOSession.Session.openClientFile (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:126458:37)
    at Session.handlers.ts.createMapFromTemplate._a.(anonymous function) (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:125591:31)
    at /Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:127131:88
    at IOSession.Session.executeWithRequestId (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:127122:28)
    at IOSession.Session.executeCommand (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:127131:33)
    at IOSession.Session.onMessage (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:127153:35)
    at Interface.<anonymous> (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/extensions/node_modules/typescript/lib/tsserver.js:128414:27)
    at Interface.emit (events.js:182:13)
    at Interface._onLine (readline.js:290:10)
    at Interface._normalWrite (readline.js:433:12)
    at Socket.ondata (readline.js:149:10)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:279:12)
    at readableAddChunk (_stream_readable.js:264:11)
    at Socket.Readable.push (_stream_readable.js:219:10)
    at Pipe.onread (net.js:636:20)
```
